packages/api/test/user.spec.js

```diff
@@ -32,8 +32,9 @@ describe('GET /user/account', () => {
     })
     assert(res.ok)
     const data = await res.json()
+    console.log('TOTAL', data.usedStorage)
     assert.strictEqual(data.usedStorage.uploaded, 32000)
-    assert.strictEqual(data.usedStorage.psaPinned, 10000)
+    assert.strictEqual(data.usedStorage.psaPinned, 710000)
```

Comment:
Needed updated as now we are counting fixtures that were not pinned status from https://github.com/web3-storage/web3.storage/blob/main/packages/api/test/fixtures/init-data.sql#L156-L167